packages/components/badge/src/badge.vue

```diff
@@ -13,8 +13,11 @@
           badgeClass,
         ]"
         :style="style"
-        v-text="content"
-      />
+      >
+        <slot name="content">
```

Comment:
Would not it be better if the content value was exported to the user?

eg. `<slot name="content" :value="content">`

Reply:
That makes sense.